scania-digital-design-system/sdds
# Dropdown fixes sprint 36

**Describe pull-request**  
Improvements on the dropdown component.
- Added arrow navigation
- Fixed disable option bug
- Increased z-index to fix overlapping  issues
- Closing dropdown when clicking on open dropdown
- Fixed bugs for inside label (Long labels doesn't align with selected option and overflows)

**Solving issue**  
[AB#1994](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1994) 
[AB#1997](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1997) 
[AB#2017](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2017) 
[AB#2014](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2014)

**How to test**  
1. Open storybook
2. Check if arrow navigation works on dropdown component (not on disabled version)
3. Try closing it by clicking on it
4. Try long labels in the inside label option and see if it works correctly
